scripts/cybershake/auto_submit.py

```diff
@@ -257,6 +264,23 @@ def submit_sl_script(script_name, **kwargs):
             ),
             target_machine=JOB_RUN_MACHINE[const.ProcessType.HF2BB].value,
         )
+    elif proc_type == const.ProcessType.plot_srf.value:
+        plot_srf_template = (
+            "--export=CUR_ENV -o {output_file} -e {error_file} {script_location} "
+            "{srf_path} {output_dir} {output_map_plot_path} {mgmt_db_loc} {run_name}"
+        )
+        script = plot_srf_template.format(
+            srf_path=sim_struct.get_srf_path(root_folder, run_name),
+            output_dir=os.path.join(verification_dir,'srf_plot'),
+            output_map_plot_path=os.path.join(sim_struct.get_srf_dir(root_folder, run_name), "{}_map.png".format(run_name)),
```

Comment:
Should have given it a clearer name like 'static_output_map_plot_path'. Because plot_srf_map.py doesn't allow specifying output dir, rather, the output is defaulted to srf dir. What I did was move the map plot from srf dir to verification dir in the end of the slurm script

Reply:
Yep. That's okay. Doesn't change my previous comment though. 😊